src/argus_htmx/templates/htmx/incidents/_incident_filterbox.html

```diff
@@ -16,6 +16,17 @@
                         <span class="label-text">{{ field.label }}</span>
                         {% if field|field_type == "booleanfield" %}
                             {{ field|add_class:"checkbox checkbox-accent border" }}
+                        {% elif field.name == "maxlevel" %}
```

Comment:
not sure if this should be the criterium for showing a slider. The cleanest way would be to use a custom widget I think, Then it can be more easily reused for other filter parameters or forms.

but probably after the demo

Reply:
I mentioned something similar in the newest commit: better to have widget templates (defined in the form widget in the form) than rely too much on widget-tweaks. After the demo :)

Reply:
See #93 for an even better solution.